include/aktualizr-lite/aklite_client_ext.h

```diff
@@ -0,0 +1,64 @@
+#ifndef AKTUALIZR_LITE_CLIENT_EXT_H_
+#define AKTUALIZR_LITE_CLIENT_EXT_H_
+
+#include <cstdlib>
+#include <string>
+
+#include "json/json.h"
+
+#include "aktualizr-lite/tuf/tuf.h"
+
+#include "aktualizr-lite/api.h"
+
+class GetTargetToInstallResult {
+ public:
+  //
+  enum class Status {
+    // First block must match CheckInResult::Status
```

Comment:
It's a pain that we have to define all `CheckInResult::Status` enum values again.